### PR TITLE
fix: issue-2553 update dingtack code

### LINKED ISSUE #2553: 钉钉的二维码过期了，能更新一下吗
**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

### PR DESCRIPTION
**Pull request by wang-mask**
Signed-off-by: wang-mask <2018091609006@std.uestc.edu.cn>

### Ⅱ. Does this pull request fix one issue?

fixes #2553 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Feel free to reach out if you have any questions. The maintainers of this projec
 DingTalk:
 
 <div>
-  <img src="https://fluid-imgs.oss-cn-shanghai.aliyuncs.com/public/imgs/dingding_code.png" width="280" title="dingtalk">
```
Comment (cheyang) [inline README.md:136]: Could you also please update [README-zh_CN.md](https://github.com/fluid-cloudnative/fluid/blob/master/README-zh_CN.md).

**Review by cheyang — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ Feel free to reach out if you have any questions. The maintainers of this projec
...
+  <img src="static/dingtalk.png" width="280" title="dingtalk">
```
Comment (wang-mask) [inline README.md:136]: @cheyang here

File: README.md (hunk 1 above)
Comment (wang-mask) [reply README.md:136]: they all refer the same picture. So [README-zh_CN.md](https://github.com/fluid-cloudnative/fluid/blob/master/README-zh_CN.md) has been update too.

Comment (cheyang) [reply README.md:136]: Thanks， I thought [README-zh_CN.md](https://github.com/fluid-cloudnative/fluid/blob/master/README-zh_CN.md) also referred  https://fluid-imgs.oss-cn-shanghai.aliyuncs.com/public/imgs/dingding_code.png.

**Pull request merged**